js/apps/filter-page/controllers/filterPageCtrl.js

```diff
@@ -14,9 +14,19 @@
 		 * @param {string} filter
 		 */
 		self.setFilter = (filter, filterFamily) => {
-			const $resultsDisplay = angular.element('#results-display');
-
 			setActiveFilters(filter, filterFamily);
+			cycleDisplay();
+		};
+
+		self.clearFilters = () => {
+			self.activeFilters = [];
+			cycleDisplay();
+		};
+
+		/* Private */
+
+		const cycleDisplay = () => {
+			const $resultsDisplay = angular.element('#results-display');
```

Comment:
Why are you using jQuery?

Reply:
I refactored to only use jquery for the event triggering. I can't use $emit() or $broadcast() since I'm talking to something outside the Angular app.